frontend/app/src/main/java/com/example/recipez/JSONArrayAdapter.java

```diff
@@ -45,14 +45,32 @@ public View getView(int position, @Nullable View convertView, @NonNull ViewGroup
         LayoutInflater layoutInflator = LayoutInflater.from(mContext);
         convertView = layoutInflator.inflate(mResource, parent, false);
 
-        ImageView imageView = convertView.findViewById(R.id.recipe_image);
-        TextView textView = convertView.findViewById(R.id.recipe_title);
+        if (objectType.equals("recipe")) {
```

Comment:
Is this what you were thinking for objectType?

Reply:
Yes ! SURELY there exists a better way to do this without having to make it case based but it'll make do for the discrepancies between each object for now